datacontract/export/bigquery_converter.py

```diff
@@ -108,6 +108,8 @@ def map_type_to_bigquery(field: Field) -> str:
         return "NUMERIC"
     elif field_type.lower() == "double":
         return "BIGNUMERIC"
+    elif field_type.lower() in ["object", "record"] and (field.fields is None or len(field.fields) == 0):
```

Comment:
Can use `not` keyword, it does the same thing
```suggestion
    elif field_type.lower() in ["object", "record"] and not field.fields:
```

Reply:
done thank u